.env copy.local

```diff
@@ -1,6 +1,7 @@
-﻿REACT_APP_BSP_ORGANIZATION=""
+REACT_APP_BSP_ORGANIZATION=""
 REACT_APP_BSP_SHARED_KEY=""
 REACT_APP_BSP_SECRET_KEY=""
 
 NEXTAUTH_URL="http://localhost:3000"
-REACT_APP_URI="http://localhost:3000"
\ No newline at end of file
+NEXTAUTH_SECRET=""
```

Comment:
can this be mentioned in the readme as a variable that needs to be set?

Reply:
Hi Nick, I already have a line in the readme to set the variable and how to set it as well.